lms/static/sass/features/_learner-profile.scss

```diff
@@ -210,133 +302,123 @@
         }
     }
 
-    .wrapper-profile-section-one {
-        @include float(left);
-        @include margin-left($baseline*3);
-        width: 300px;
-        background-color: $white;
-        border-top: 5px solid $blue;
-        padding-bottom: $baseline;
-
+    .wrapper-profile-section-container-one {
         @media (max-width: $learner-profile-container-flex) { // Switch to map-get($grid-breakpoints,md) for bootstrap
-            @include margin-left(0);
-            width: 100%;
+            width: 90%;
+            padding: 0 5%;
         }
-    }
-
-    .profile-section-one-fields {
-        @include margin(0, $baseline/2, 0, $baseline*0.75);
 
-        .social-links {
-            font-size: 2rem;
-            padding-top: $baseline/4;
+        .wrapper-profile-section-one {
+            @include float(left);
+            @include margin-left($baseline*3);
+            width: 300px;
+            background-color: $white;
+            border-top: 5px solid $blue;
+            padding-bottom: $baseline;
 
-            & > span {
-                color: $gray-l4;
+            @media (max-width: $learner-profile-container-flex) { // Switch to map-get($grid-breakpoints,md) for bootstrap
+                @include margin-left(0);
+                width: 100%;
             }
 
-            a {
```

Comment:
This needs to be added back too, if you are rebased off of master, you can make sure that the links still show up as expected